blueflood-core/src/main/java/com/rackspacecloud/blueflood/inputs/constraints/EpochRange.java

```diff
@@ -0,0 +1,50 @@
+package com.rackspacecloud.blueflood.inputs.constraints;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+import javax.validation.ReportAsSingleViolation;
+import javax.validation.constraints.Min;
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.*;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+@Target({ METHOD, FIELD, ANNOTATION_TYPE, CONSTRUCTOR, PARAMETER })
+@Retention(RUNTIME)
+@Documented
+@Constraint(validatedBy = { EpochRangeValidator.class })
+@ReportAsSingleViolation
+public @interface EpochRange {
```

Comment:
maybe a small comment how most of this is boilerplate from hibernate validator?  At first look, it kinda freaks me out.  You're call.  lgtm